tools/Makefile

```diff
@@ -0,0 +1,59 @@
+GLIDE = github.com/tendermint/glide
+TOOLS_CHECK = glide
+
+all: install_glide check get_vendor_deps install
+
+
+########################################
+### Glide
+
+check:
+	@# https://stackoverflow.com/a/25668869
+	@echo "Found tools: $(foreach tool,$(TOOLS_CHECK),\
+        $(if $(shell which $(tool)),$(tool),$(error "No $(tool) in PATH")))"
+
+install_glide:
+	@echo "$(ansi_grn)Installing glide$(ansi_end)"
+	go get -u -v $(GLIDE)
```

Comment:
Let's remove `-u` since updating glide is a specific step.

Reply:
I dunno, if they already have glide, it's better to update it?

Reply:
I'll add a test for existence of glide as well.